module/VuFind/src/VuFind/Search/QueryAdapter.php

```diff
@@ -106,6 +107,11 @@ public static function display(AbstractQuery $query, $translate, $showName)
             return $query->getString();
         }
 
+        // Work keys query:
+        if ($query instanceof WorkKeysQuery) {
+            return $query->getId();
```

Comment:
Is it an issue that this might return null? Do we need some kind of default fallback (even if it's just an empty string)?

Reply:
Yes, fallback added.